src/components/FilterList.tsx

```diff
@@ -0,0 +1,30 @@
+import { ChangeEvent } from "react";
+
+interface FilterListProps {
+	searchTerm: string;
+	setSearchTerm: (term: string) => void;
+}
+
+export function FilterList({ searchTerm, setSearchTerm }: FilterListProps) {
+	const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+		setSearchTerm(e.target.value);
+	};
+
+	return (
+		<>
```

Comment:
Minor! I noticed the use of the fragments <> </> is not needed since there is only one higher level parent element , the form element. :)

Reply:
Good eye!

Reply:
Very good catch thank you!